src/ATokenFlexVoting.sol

```diff
@@ -2,6 +2,8 @@
 pragma solidity 0.8.10;
 
 // forgefmt: disable-start
+import {IFractionalGovernor} from "src/interfaces/IFractionalGovernor.sol";
+import {IVotingToken} from "src/interfaces/IVotingToken.sol";
```

Comment:
Nit on sort order, `src` should be last, item 2 here: https://book.getfoundry.sh/tutorials/best-practices#general-contract-guidance (should add this to scopelint)